http/src/main/java/com/redhat/lightblue/client/http/LightblueHttpClient.java

```diff
@@ -76,21 +80,19 @@ public String getRestURI(String baseServiceURI) {
             if (!baseServiceURI.endsWith("/")) {
                 b.append('/');
             }
-            b.append(uri);
+            b.append("lock/");
```

Comment:
I don't think its needed, but saw it elsewhere so probably just being consistent